### ISSUE TITLE
IPV6 Support

### ISSUE THREAD

**Issue by attiqueullah** [VoIPGRID/VialerSIPLib#112]
### Version

{version or tag here}

### File / Feature

{file or feature containing the issue}

### Expected behaviour

{what should happen}

### Actual behaviour

{what happens}

### Stacktrace / Error message

{paste here}

### Other info

{anything else that might be related/useful}

**Comment by redmerloen** [comment]
Vialer-pjsip-ios 3.3.0 has been released with IP v6 support. Soon this library will be updated to that version. And then it should work if you pass the correct TransportConfiguration

**Comment by attiqueullah** [comment]
Thanks for reply . It is great to hear from your side . What type of TransportConfiguration i should use for ipv6 in your library?

**Comment by redmerloen** [comment]
This library will be updated soon.
VSLTransportConfigurationUDP6 or VSLTransportConfigurationTLS6 or VSLTransportConfigurationTCP6

**Comment by attiqueullah** [comment]
has the library updated to 3.3.0?

**Comment by redmerloen** [comment]
yes, but i suspect there is maybe more needed to make it work with ipv6. not sure because i can't test on ipv6

**Comment by tnmendes** [comment]
Apple is not accepting apps that do not support IPv6 :-/

@redmerloen if you have macbook with wireless and ethernet you can test IPv6.

https://developer.apple.com/library/content/documentation/NetworkingInternetWeb/Conceptual/NetworkingOverview/UnderstandingandPreparingfortheIPv6Transition/UnderstandingandPreparingfortheIPv6Transition.html#//apple_ref/doc/uid/TP40010220-CH213-SW1

**Comment by jeremynorman89** [comment]
Hey, just to provide a quick update on this issue:

While we understand the importance of IPv6 support, it's not something that we currently have demand for and therefore is not high priority for us to resolve currently. I have also seen that Apple is meant to reject apps that don't support IPv6 but that was meant to start June 2016, so nearly three years ago, and none of our releases have been rejected for this reason.

We hope to do some internal testing (using the method provided above, thank you @tnmendes) in the next month or so and based on this, try and determine how much work will be needed to properly support IPv6 and whether we can justify the time required.

In the meantime, we would absolutely welcome any PRs that resolve issues regarding IPv6.
